grammars/Kotlin.tmLanguage.json

```
@@ -522,9 +525,41 @@
             "match": "\\bnull\\b",
             "name": "constant.language.null.kotlin"
         },
+        "lambda-expression": {
```

Since this doesn't match complete lambda expressions, just the arrow syntax, we might want to consider naming this differently, e.g.

```suggestion
        "lambda-arrow": {
```

---

Good call. I've updated it.